City-of-Helsinki/youth-membership-admin-ui
# YM-419 | Release/1.2.0

## Description
This PR contains the latest changes from develop branch.

## Context

[YM-419](https://helsinkisolutionoffice.atlassian.net/browse/YM-419)

## How Has This Been Tested?

The changes contain only automatic tests which have been run.

## Manual Testing Instructions for Reviewers

1) Add values to `.env` file (if you don't have these you can contact Santtu):
- `REACT_APP_TESTING_USERNAME`
- `REACT_APP_TESTING_PASSWORD`
- `REACT_APP_TESTING_USERNAME_WITH_PROFILE`
- `REACT_APP_MAILOSAUR_API`
- `REACT_APP_MAILOSAUR_SERVER_ID`
2) Login to [profile-admin](https://profiili-api.test.kuva.hel.ninja/admin) and make sure "Uno User" has no profile
3) Start local application `yarn start`
4) Run `yarn browser-test`
5) Delete the created profile (Uno User) to avoid crashing next time tests are run. **DO NOT DELETE Existing TestProfile**

## Screenshots

N/A
